ui/values/localizable/en.go

```diff
@@ -772,7 +772,7 @@ const EN = `
 "wouldTradeCex" = "Would you like to trade on CEX?"
 "writeDownAllXWords" = "Write down all %d words in the correct order."
 "writeDownSeed" = "Write down seed phrase"
-"wroteAllWords" = "I have written down all 33 words"
+"wroteAllWords" = "I have written down all words"
```

Comment:
I was expecting this to be dynamic but good.

Reply:
this should be dynamic